cn.lpkt.serverbox.app.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version='1.0' encoding='utf-8'?>
+<component type="desktop-application">
+  <!--Created with jdAppStreamEdit 9.1-->
+  <id>cn.lpkt.serverbox.app</id>
+  <name>ServerBox</name>
+  <summary>ServerBox - server status &amp; toolbox</summary>
+  <developer id="lollipopkit">
```

Comment:
must be reverse domain

Reply:
`cn.lpkt.serverbox.app` is a reverse domain

Reply:
that's not what the developer id is set to.